### PR TITLE
[JBPM-9317] Add test for springboot userTaskService.saveContent

### PR DESCRIPTION
**Pull request by gmunozfe**
**JIRA**:  [JBPM-9317](https://issues.redhat.com/browse/JBPM-9317)

Added springboot test for checking saveContent accesibility during deadline notification

### CODE REVIEW

**Comment by gmunozfe** [comment]
jenkins retest this

File: kie-spring-boot/kie-spring-boot-samples/jbpm-spring-boot-sample-basic/src/test/java/org/jbpm/springboot/samples/DeadlinesNotificationTest.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    @Test(timeout = 10000)
+    public void testSaveContentAfterDeadlines() throws Exception {
+        countDownListener.configure(1);
+        countDownListener.setSaveContent(true);
+
+        assertNotNull(unit);
+
+        long processInstanceId = processService.startProcess(unit.getIdentifier(), "notification");
+
+        assertNotNull(processInstanceId);
```
Comment (elguardian) [inline kie-spring-boot/kie-spring-boot-samples/jbpm-spring-boot-sample-basic/src/test/java/org/jbpm/springboot/samples/DeadlinesNotificationTest.java:100]: Cannot never be null

**Review by elguardian — CHANGES_REQUESTED** [review]
some change otherwise LGTM

File: kie-spring-boot/kie-spring-boot-samples/jbpm-spring-boot-sample-basic/src/test/java/org/jbpm/springboot/samples/DeadlinesNotificationTest.java (hunk 1 above)
Comment (gmunozfe) [reply kie-spring-boot/kie-spring-boot-samples/jbpm-spring-boot-sample-basic/src/test/java/org/jbpm/springboot/samples/DeadlinesNotificationTest.java]: OK, fixed, thanks for the review!

**Review by elguardian — APPROVED** [review]

**Comment by elguardian** [comment]
thx @gmunozfe

**Pull request merged**